pkg/podvolume/util.go

```diff
@@ -48,10 +50,33 @@ const (
 	InitContainer = "restic-wait"
 )
 
+// VolumeBackupInfo describes the backup info of a volume backed up by PodVolumeBackups
```

Comment:
If this is not intended to be used by other packages we don't want to expose it.

Reply:
Looks like this is for back compatibility of very old version, let's double check it and also discuss it in the community meeting, if we finally confirm it is not used any more, let's remove it.
As the discussion, we will not change it in the current PR, and will open an issue to trace it.

Reply:
Done, thanks.